test/createVM/main.go

```diff
@@ -0,0 +1,298 @@
+package main
+
+import (
+	"bytes"
+	"embed"
+	"flag"
+	"fmt"
+	"os"
+	"text/template"
+
+	"github.com/dypflying/go-qcow2lib/qcow2"
+	bmoe2e "github.com/metal3-io/baremetal-operator/test/e2e"
+	"libvirt.org/go/libvirt"
```

Comment:
Could you please use test this syntax?:
 
```suggestion
	"github.com/libvirt/libvirt-go-module"
```